python/dronin/uavo_collection.py

```diff
@@ -155,3 +155,47 @@ def from_uavo_xml_path(self, path):
                 content_list.append(f.read())
 
         self.from_file_contents(content_list)
+
+    # These XML methods are not in an ideal place, but better than
+    # anywhere else I could think of.  Plunk them down here for now.
+    @classmethod
+    def _prettify(cls, elem):
+        """Return a pretty-printed XML string for the Element.
+        """
+        from xml.dom import minidom
+        try:
+            import lxml.etree as etree
+        except:
+            import xml.etree.ElementTree as etree
+
+        rough_string = etree.tostring(elem)
+        reparsed = minidom.parseString(rough_string)
+        return reparsed.toprettyxml(indent="  ")
+
+    @classmethod
+    def ExportXML(cls, objects, githash=None):
```

Comment:
[Can we not have uppercase methods please](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)? :wink:

Reply:
k